edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -143,7 +143,10 @@ where
         future::ok(self)
             .and_then(Self::write_check)
             .and_then(Self::write_module_logs)
-            .and_then(Self::write_edgelet_log)
+            .and_then(|s| s.write_system_log("aziot-keyd", "aziot-keyd"))
```

Comment:
Before the MR refactory and integrating with iot-identity-service repo. done, I'd recommend we align this effort with MR refactory in the IIOT repo. 

Then we can link the two repo, verify and merge to master.

Reply:
Rebased this PR on the `iiot` branch where module runtime refactoring is happening